src/core/logging_manager.py

```diff
@@ -0,0 +1,129 @@
+"""Logging Manager.
+
+This module provides classes to represent logging configuration.
+"""
+
+import json
+import logging
+import os
+import sys
+from datetime import datetime
+
+import colorlog
+
+from src.core.constants import CommonPaths
+
+
+class PrintHandler(logging.StreamHandler):
+    """Custom Print handler."""
+
+    def emit(self, record):
+        """Custom print handler.
+
+        :param record:
+        :return:
+        """
+        msg = self.format(record)
+        print(msg)
+        sys.stdout.flush()
+        sys.stderr.flush()
+
+
+class CustomJsonFormatter(logging.Formatter):
+    """Custom JSON formatter."""
+
+    def format(self, record: logging.LogRecord):
+        """Custom json format for log message.
+
+        :param record:
+        :return:
+        """
+        log_message = {
+            "datetime": datetime.now().strftime("%Y-%m-%d %H:%M:%S"),
+            "name": record.name,
+            "level": record.levelname,
+            "filename": f"{record.filename}:{record.lineno}",
+            "message": record.getMessage(),
+        }
```

Comment:
@vasylboliuk OPTIONAL: it could be useful to move this configuration to your config file to make it tweakable and configurable